app/components/dns-records-table.tsx

```diff
@@ -121,7 +80,6 @@ export default function DnsRecordsTable(props: DnsRecordsTableProps) {
           <Table variant="striped" colorScheme="gray">
             <Thead>
               <Tr>
-                <Th />
                 <Th>Subdomain</Th>
```

Comment:
I think we should change this column name to DNS Record

Reply:
Good idea, I'll do that.